yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
What did you do to this file?

Reply:
I don't recall doing anything to this file. It's automatically generated when we start up the server and it doesn't exist or changes are made to the package.json file correct?

Reply:
So this file should not be here.  This yarn.lock is at the root of the project, you must have run yarn in the wrong directory.  Please remove it and add back the proper yarn.lock file in `src/main/webapp`

Reply:
Whoops. Fixed in 5acaf2331854e9b3bd7deed5605d5d44f3cd1fd7